server/routes/visitJourney/checkYourBooking.ts

```diff
@@ -18,6 +18,7 @@ export default class CheckYourBooking {
     const isUpdate = this.mode === 'update'
     const { visitSessionData } = req.session
     const { offenderNo } = visitSessionData.prisoner
+    const prisonerName = `${visitSessionData.prisoner.firstName} ${visitSessionData.prisoner.lastName}`
```

Comment:
In the routes changed in this PR it looks like there are two different approaches being used to get prisoner name to a template:
1. build an (additional) `prisonerName` in the route and pass this to template to render as `{{ prisonerName }}`
2. pass, say, `prisoner` to the template and then render as `{{ prisoner.firstName + " " + prisoner.lastName }}`

Either is fine but it's probably worth being consistent. My slight preference would be that if the data is going to the template anyway then keep the route simpler by not creating an extra variable - and this keeps the presentational logic in the view.  Not much in it though: consistency is the main thing.